docs/reference/buildx_create.md

```diff
@@ -151,6 +151,13 @@ No driver options.
 #### `docker-container` driver
 
 - `image=IMAGE` - Sets the container image to be used for running buildkit.
+- `memory=MEMORY` - Sets the container memory limit to be used for running buildkit.
+- `memory-swap=MEMORY_SWAP` - Sets the container memory swap limit to be used for running buildkit.
+- `cpu-quota=CPU_QUOTA` - Sets the CPU CFS (Cmpletely Fair Scheduler) quota limit to be used for running buildkit.
+- `cpu-period=CPU_PERIOD` - Sets the CPU CFS (Cmpletely Fair Scheduler) period limit to be used for running buildkit.
+- `cpu-shares=CPU_SHARES` - Sets the CPU shares (relative weight) limit to be used for running buildkit.
+- `cpuset-cpus=CPUSET_CPUS` - Sets the CPU cores to be used for running buildkit, such as (0-3, 0,1).
+- `cpuset-mems=CPUSET_MEMS` - Sets the CPU memory nodes to be used for running buildkit, such as (0-3, 0,1).
 - `network=NETMODE` - Sets the network mode for running the buildkit container.
 - `cgroup-parent=CGROUP` - Sets the cgroup parent of the buildkit container if docker is using the "cgroupfs" driver. Defaults to `/docker/buildx`.
```

Comment:
I'd suggest the following edits, and adding a link to the relevant page in the engine docs.

```suggestion
- `image=IMAGE` - Sets the BuildKit image to use for the container.
- `memory=MEMORY` - Sets the amount of memory the container can use.
- `memory-swap=MEMORY_SWAP` - Sets the memory swap limit for the container.
- `cpu-quota=CPU_QUOTA` - Imposes a CPU CFS quota on the container.
- `cpu-period=CPU_PERIOD` - Sets the CPU CFS scheduler period for the container.
- `cpu-shares=CPU_SHARES` - Configures CPU shares (relative weight) of the container.
- `cpuset-cpus=CPUSET_CPUS` - Limits the set of CPU cores the container can use.
- `cpuset-mems=CPUSET_MEMS` - Limits the set of CPU memory nodes the container can use.
- `network=NETMODE` - Sets the network mode for the container.
- `cgroup-parent=CGROUP` - Sets the cgroup parent of the container if docker is using the "cgroupfs" driver. Defaults to `/docker/buildx`.

Before you configure the resource limits for the container, read about [configuring runtime resource constraints for containers](https://docs.docker.com/config/containers/resource_constraints/).
```